pkgs/applications/misc/plots/default.nix

```diff
@@ -0,0 +1,78 @@
+{ fetchFromGitHub
+, fetchzip
+, gobject-introspection
+, lib
+, libadwaita
+, python3Packages
+, python3
+, wrapGAppsHook4
+}:
+let
+  latinmodern-math = fetchzip {
+    url = "https://www.gust.org.pl/projects/e-foundry/lm-math/download/latinmodern-math-1959.zip";
+    hash = "sha256-et/WMhfZZYgP0S7ZmI6MZK5owv9bSoMBXFX6yGSng5Y=";
+  };
+in
+python3Packages.buildPythonApplication rec {
+  pname = "plots";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "alexhuntley";
+    repo = "Plots";
+    rev = "v${version}";
+    hash = "sha256-GjNpaorxkkhZsqrKq4kO5nqF5+4I4tmSc023AZpY8Sw=";
+  };
+
+  propagatedNativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook4
```

Comment:
I think wrapGAppsHook4 is most often put in nativeBuildInputs. What's the reason to put it in propagatedNativeBuildInputs here?

Reply:
gobject-introspection is also not in propagated* in this example: https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/gnome.section.md#when-using-wrapgappshook-with-special-derivers-you-can-end-up-with-double-wrapped-binaries-ssec-gnome-common-issues-double-wrapped